src/handlers/records-read.ts

```diff
@@ -37,13 +37,9 @@ export class RecordsReadHandler implements MethodHandler {
       return messageReplyFromError(e, 401);
     }
 
-    // get existing messages matching `recordId` so we can perform authorization
-    const query = {
-      interface : DwnInterfaceName.Records,
-      recordId  : message.descriptor.recordId
-    };
+    // get existing messages matching `recordId` or `protocol` with `protocolPath` so we can perform authorization
+    const query: Filter = RecordsReadHandler.createFilter(recordsRead.message.descriptor);
     const existingMessages = await this.messageStore.query(tenant, query) as TimestampedMessage[];
-
     const newestExistingMessage = await Message.getNewestMessage(existingMessages);
```

Comment:
This will return the most recently _updated_ record, not the most recently created record. Is this the expected behavior?

Reply:
Good call out, this was my understanding of the intended behavior, do you think that's unclear?

When you think about using this feature, do you find yourself thinking of scenarios that latest Created should be returned instead of updated?

Reply:
I was expecting the latest created to be returned tbh. What's the rationale for returning the latest updated vs latest created?

@csuwildcat What say you? To borrow/adapt one of your examples, I imagine someone has a blog that they add new blog posts to. They want readers to be able to `RecordsRead` `protocolPath: 'blog/post'` to get the latest created blog post. If the blog owner makes a spelling fix to an older post, they wouldn't want that older post to be returned.